src/documents/ModelRelay.md

```diff
@@ -134,20 +145,48 @@ This is done using the `modelRelay` section of a component's top-level options.
         </tr>
```

Comment:
It is to some extend supported since it is tested here: https://github.com/fluid-project/infusion/blob/master/tests/framework-tests/core/js/DataBindingTests.js#L1154 - I don't know of a case where it is being used in the wild and we do somewhat discourage it - see IRC discussion at https://botbot.me/freenode/fluid-work/2016-04-07/?msg=63718987&page=2

We expect most cases that might currently require nested transforms to be taken care of by upcoming work on the REDUCED EXPRESSION PARSER - https://issues.fluidproject.org/browse/FLUID-5894 - the facility for nested transforms in relays is left around in "hopeful exploration" to see if anyone can claim a use for it.